openhtf/util/validators.py

```diff
@@ -87,6 +87,10 @@ def __init__(self, minimum=None, maximum=None):
       self.maximum = maximum
 
     def __call__(self, value):
+      # Check for nan
+      import math
```

Comment:
He makes a good point, the only way to intercept this the way things are is to override `__getattribute__` in `Validators`. I think this is the best place to do the import, considering.